code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,235 @@
 	icon = 'icons/obj/eldritch.dmi'
 	icon_state = "eldrich_flask"
 	list_reagents = list(/datum/reagent/eldritch = 50)
+
+/obj/item/clothing/head/hooded/cult_hoodie/void
+	name = "void hood"
+	icon_state = "void_cloak"
+	flags_inv = NONE
+	flags_cover = NONE
+	desc = "Black like tar, doesn't reflect any light. Runic symbols line the outside, with each flash you loose comprehension of what you are seeing."
+	item_flags = EXAMINE_SKIP
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30,ENERGY = 30, BOMB = 15, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
+
+/obj/item/clothing/suit/hooded/cultrobes/void
+	name = "void cloak"
+	desc = "Black like tar, doesn't reflect any light. Runic symbols line the outside, with each flash you loose comprehension of what you are seeing."
+	icon_state = "void_cloak"
+	inhand_icon_state = "void_cloak"
+	allowed = list(/obj/item/melee/sickly_blade, /obj/item/forbidden_book, /obj/item/living_heart)
+	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/void
+	flags_inv = NONE
+	// slightly worse than normal cult robes
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30,ENERGY = 30, BOMB = 15, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/void_cloak
+	///reference to the componenet so we dont have to call get_component every time we toggle hood
+	var/datum/component/storage/concrete/pockets/void_cloak/cloak
+	alternative_mode = TRUE
+
+/obj/item/clothing/suit/hooded/cultrobes/void/ToggleHood()
+	if(!iscarbon(loc))
+		return
+	var/mob/living/carbon/carbon_user = loc
+	if(IS_HERETIC(carbon_user) || IS_HERETIC_MONSTER(carbon_user))
+		. = ..()
+		//We need to account for the hood shenanigans, and that way we can make sure items always fit, even if one of the slots is used by the fucking hood.
+		if(suittoggled)
+			to_chat(carbon_user,"<span class='notice'>The light shifts around you making the cloak invisible!</span>")
+		else
+			to_chat(carbon_user,"<span class='notice'>The kaleidoscope of colours collapses around you, as the cloak shifts to visibility!</span>")
+		item_flags = suittoggled ? EXAMINE_SKIP : ~EXAMINE_SKIP
+	else
+		to_chat(carbon_user,"<span class='danger'>You can't force the hood onto your head!</span>")
+
+
+/obj/item/clothing/mask/void_mask
+	name = "Abyssal Mask"
+	desc = "Mask created from the suffering of existance, you can look down it's eyes, and notice something gazing back at you."
+	icon_state = "mad_mask"
+	inhand_icon_state = "mad_mask"
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	flags_inv = HIDEFACE|HIDEFACIALHAIR
+	var/mob/living/carbon/human/local_user
+
+/obj/item/clothing/mask/void_mask/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user) && user.mind && slot == ITEM_SLOT_MASK)
+		local_user = user
+		START_PROCESSING(SSobj,src)
+		if(IS_HERETIC(user) || IS_HERETIC_MONSTER(user))
+			return
+		ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
```

Comment:
```suggestion
		else
			ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
```
This way you won't need the removal check at 263 since we're already returning earlier.